src/Inciter/Discretization.cpp

```diff
@@ -506,6 +521,13 @@ Discretization::status()
   // Query after how many time steps user wants TTY dump
   const auto tty = g_inputdeck.get< tag::interval, tag::tty >();
 
+  // estimate grind time (taken between this and the previous time step)
+  using std::chrono::duration_cast;
+  using ms = std::chrono::milliseconds;
+  using clock = std::chrono::high_resolution_clock;
+  auto grind_time = duration_cast< ms >(clock::now() - m_prevstatus).count();
+  m_prevstatus = clock::now();
```

Comment:
Note to myself: This chunk of code really only need to be done for chareid 0, since `m_prevstatus` is only used by that, but I guess it does not really hurt much.